packages/zoe/src/types.js

```diff
@@ -335,12 +336,20 @@
  */
 
 /**
+ * @typedef {Object} StandardTerms
+ * @property {IssuerKeywordRecord} terms.issuers - record with
+ * keywords keys, issuer values
+ * @property {BrandKeywordRecord} terms.brands - record with keywords
+ * keys, brand values
+ * @property {AmountMathKeywordRecord} terms.maths - record with keywords
+ * keys, amountMath values
+ * 
+ * @typedef {StandardTerms & Record<string, any>} Terms
```

Comment:
I don't think you want a conjunctive type here, but rather an `extends`. But this remains beyond my expertise. @michaelfig ?

Reply:
This was recommended by Michael a week or so ago. It may be outdated :D

Reply:
Nope, `extends` is not available in JSDoc, so intersection it is!